docs/conf.py

```diff
@@ -2,306 +2,172 @@
-     [AUTHOR_NAME], 1)
+    ('index', project.lower(), '{} Documentation'.format(project),
+     [author], 1)
 ]
 
-# If true, show URL addresses after external links.
-# man_show_urls = False
+# -- Options for Texinfo output --------------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-texinfo-output
 
-
-# -- Options for Texinfo output -------------------------------------------
-
-# Grouping the document tree into Texinfo files. List of tuples
-# (source start file, target name, title, author,
-#  dir menu entry, description, category)
 texinfo_documents = [
-    ('index', PACKAGE_NAME, u'{} Documentation'.format(PACKAGE_TITLE),
-     AUTHOR_NAME, PACKAGE_TITLE, 'One line description of project.',
+    ('index', project.lower(), '{} Documentation'.format(project),
+     author, project, 'One line description of project.',
      'Miscellaneous'),
 ]
 
-# Documents to append as an appendix to all manuals.
-# texinfo_appendices = []
+# -- Options for intersphinx -----------------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#configuration
 
-# If false, no module index is generated.
-# texinfo_domain_indices = True
-
-# How to display URL addresses: 'footnote', 'no', or 'inline'.
-# texinfo_show_urls = 'footnote'
+intersphinx_mapping = {
+    "python": ("https://docs.python.org/3", None),
+}
 
-# If true, do not generate a @detailmenu in the "Top" node's menu.
-# texinfo_no_detailmenu = False
+# -- Options for Autodoc --------------------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#configuration
 
+# Both the class’ and the __init__ method’s docstring are concatenated and inserted.
+# Pybind11 generates class signatures on the __init__ method.
+autoclass_content = "both"
 
-def run_apidoc(_):
-    """This method is required by the setup method below."""
-    ignore_paths = ['opentimelineio_contrib.adapters', 'tests', 'setup.py']
-    # https://github.com/sphinx-doc/sphinx/blob/master/sphinx/ext/apidoc.py
-    argv = [
-        '--force',
-        '--no-toc',
-        # '--separate',
-        '--module-first',
-        '--output-dir',
-        './api/modules',
-        # this is expected to be run from the `docs` directory.  Its possible
-        # to set it up to run from other places, but so far this has worked for
-        # us
-        '../src/py-opentimelineio/opentimelineio',
-    ] + ignore_paths
+autodoc_default_options = {
+    'undoc-members': True
+}
 
-    from sphinx.ext import apidoc
-    apidoc.main(argv)
+# -- Options for MySt-Parser ---------------------------------------------------------------
+# https://myst-parser.readthedocs.io/en/latest/sphinx/reference.html
+
+myst_heading_anchors = 5
+
+# -- Custom ----------------------------------------------------------------------------
+
+def process_signature(
```

Comment:
does the need for process_signature disappear completely when 2621 lands? or just the identified chunk below?

Reply:
Just the identified block (https://github.com/PixarAnimationStudios/OpenTimelineIO/pull/1209/files#diff-85933aa74a2d66c3e4dcdf7a9ad8397f5a7971080d34ef1108296a7c6b69e7e3R125-R144). Removing `self` is still something that is not yet talked about in Pybind11 issues.